src/core.ts

```diff
@@ -284,43 +286,43 @@ export class ProcessPromise extends Promise<ProcessOutput> {
         },
         // prettier-ignore
         end: (data, c) => {
-          self._resolved = true
-          const { error, status, signal, duration, ctx } = data
-          const { stdout, stderr, stdall } = ctx.store
-          const dto: ProcessOutputLazyDto = {
-            // Lazy getters
-            code: () => status,
-            signal: () => signal,
-            duration: () => duration,
-            stdout: once(() => stdout.join('')),
-            stderr: once(() => stderr.join('')),
-            stdall: once(() => stdall.join('')),
-            message: once(() => ProcessOutput.getExitMessage(
-              status,
-              signal,
-              dto.stderr(),
-              self._from
-            )),
-            ...error && {
-              code: () => null,
-              signal: () => null,
-              message: () => ProcessOutput.getErrorMessage(error, self._from)
+            self._resolved = true
```

Comment:
Let's keep the formatting for valid git blame.

Reply:
Done

Reply:
**core.ts**
<img width="438" alt="image" src="https://github.com/user-attachments/assets/44cd3484-b49e-48e4-887e-b7be4d7e48dc" />

Reply:
Sorry, I rebase another commit, everything is ok now